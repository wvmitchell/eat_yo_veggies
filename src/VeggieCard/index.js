import React from 'react'

const VeggieCard = (props) => {
  return(
    <div className="veggieCard">
      {props.name}
    </div>
  )
}

export default VeggieCard
