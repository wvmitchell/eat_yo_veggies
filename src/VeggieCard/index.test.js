import React from 'react'
import { shallow } from 'enzyme'
import VeggieCard from './index'

describe('VeggieCard', () => {
  it('should match the snapshot', () => {
    // Setup && Execution
    const wrapper = shallow(<VeggieCard name="kale" />)

    // Expectation
    expect(wrapper).toMatchSnapshot()
  })
})
