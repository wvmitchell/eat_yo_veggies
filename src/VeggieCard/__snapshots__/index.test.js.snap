// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VeggieCard should match the snapshot 1`] = `
<div
  className="veggieCard"
>
  kale
</div>
`;
